Tests/LibWeb/Text/input/performance-now.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<script src="include.js"></script>
+<script>
+    test(() => {
+        let before = performance.now();
+        let event = new window["MouseEvent"]('test');
```

Comment:
It should be fine to use `new MouseEvent('test')` here.

Reply:
Fixed 👍